keyboards/ergodox_ez/matrix.c

```diff
@@ -193,42 +124,47 @@ uint8_t matrix_key_count(void) {
  * pin: B5  B4  B3  B2  B1  B0
  */
 static void init_cols(void) {
-  // init on mcp23018
-  // not needed, already done as part of init_mcp23018()
-
-  // init on teensy
-  // Input with pull-up(DDR:0, PORT:1)
-  DDRF &= ~(1 << 7 | 1 << 6 | 1 << 5 | 1 << 4 | 1 << 1 | 1 << 0);
-  PORTF |= (1 << 7 | 1 << 6 | 1 << 5 | 1 << 4 | 1 << 1 | 1 << 0);
+    // init on mcp23018
+    // not needed, already done as part of init_mcp23018()
+
+    // init on teensy
+    setPinInputHigh(F0);
+    setPinInputHigh(F1);
+    setPinInputHigh(F2);
+    setPinInputHigh(F3);
+    setPinInputHigh(F4);
+    setPinInputHigh(F5);
+    setPinInputHigh(F6);
+    setPinInputHigh(F7);
```

Comment:
Why all the pins? Seems like F2 and F3 aren't used.

Reply:
Whoops. 
```suggestion
    // init on teensy
    setPinInputHigh(F0);
    setPinInputHigh(F1);
    setPinInputHigh(F4);
    setPinInputHigh(F5);
    setPinInputHigh(F6);
    setPinInputHigh(F7);
```